lib/jenkins_pipeline_builder/extension_dsl.rb

```diff
@@ -17,6 +17,6 @@ def job_attribute(&block)
 
   JenkinsPipelineBuilder.registry.register([:job], set)
   versions = set.extensions.map(&:min_version)
-  puts "Successfully registered #{set.name} for versions #{versions}" if set.announced
+  JenkinsPipelineBuilder.logger.info "Successfully registered #{set.name} for versions #{versions}" if set.announced
```

Comment:
There isn't a logger method defined in this file, so the other one should probably be JenkinsPipelineBuilder.logger.